changelogs/add-7373-slotfill-task-api

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: Add
+
+Adding Slotfill support to new API-driven tasks components
```

Comment:
I think it might be a good idea to have a single changelog entry for the new task API features.  I don't have a strong opinion on this, but my feeling is that this could be a confusing entry when the feature is not yet turned on.

Reply:
True, glad to remove this here.